src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -905,3 +905,68 @@ Usage example:
       })
     }
   }
+
+Error Handling in Python
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Good error handling is a key requirement in creating a good user experience
+and providing a good API.
+
+Dashboard code should not duplicate C++ code. Thus, if error handling in C++
```

Comment:
Just a suggestion:

"The Dashboard code should not duplicate the C++ code. Thus, if the error handling in C++ is sufficient, we shouldn't build your own wrapper. On the other hand, input validation is the best place to catch errors and generate the best error messages. If required, generate errors as soon as possible."

Reply:
fixed.